kubernetes/config/kube_config.c

```diff
@@ -110,18 +110,20 @@ static char *getWorkingConfigFile(const char *configFileNamePassedIn)
     if (configFileNamePassedIn) {
         configFileName = strdup(configFileNamePassedIn);
     } else {
-#ifndef _WIN32
-        kubeconfig_env = secure_getenv(ENV_KUBECONFIG);
-#else
+
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__) || defined(__APPLE__)
```

Comment:
Why does the above code use `#ifdef __linux` but this code uses a different macro `#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__) || defined(__APPLE__)` ?

Reply:
Because before this PR I thought that every OS except Windows uses secure_getenv(), but now I know that Windows and MacOS use getenv(), Linux uses secure_getenv(), and mobile platforms (probably) don't have either of these.

Reply:
I suggest keeping consistent in the two places (incluster_config.c and kube_config.c) no matter what the macro is.

Reply:
done